.buildkite/tests.sh

```diff
@@ -6,6 +6,9 @@ trap 'kill $(jobs -p); wait' SIGINT SIGTERM
 
 export EARTHLY_CONVERSION_PARALLELISM=5
 
+EARTHLY_VERSION_FLAG_OVERRIDES="$(tr -d '\n' < .earthly_version_flag_overrides)"
+export EARTHLY_VERSION_FLAG_OVERRIDES
```

Comment:
```suggestion
export EARTHLY_VERSION_FLAG_OVERRIDES="$(tr -d '\n' < .earthly_version_flag_overrides)"
```

Reply:
I had to break it up in order to have error checking for a case where the `.earthly_version_flag_overrides` file doesn't exist:

```
#!/bin/bash
set -euo pipefail
export EARTHLY_VERSION_FLAG_OVERRIDES="$(tr -d '\n' < this-file-doesnt-exist)"
echo "EARTHLY_VERSION_FLAG_OVERRIDES was set to $EARTHLY_VERSION_FLAG_OVERRIDES"


EARTHLY_VERSION_FLAG_OVERRIDES="$(tr -d '\n' < this-file-doesnt-exist)"
export EARTHLY_VERSION_FLAG_OVERRIDES
echo "this shouldnt be called"
```

When run this will display:
```
$ ./break.sh 
./break.sh: line 3: this-file-doesnt-exist: No such file or directory
EARTHLY_VERSION_FLAG_OVERRIDES was set to 
./break.sh: line 7: this-file-doesnt-exist: No such file or directory
```

showing that the all-in-one command doesn't fail when the file doesn't exist.

I initial discovered this via: https://github.com/koalaman/shellcheck/wiki/SC2155

Reply:
ah gotcha